src/main/webapp/index.html

```diff
@@ -35,18 +34,21 @@ <h4 class="search-title">Filter the Graph</h4>
     <!-- Box for zooming in -->
     <div id="zoom-box" class="input-child">
       <h4 class="search-title">Zoom In</h4>
-        <label>Zoom on Node in Graph: </label>
-        <input type="text" id="node-search" placeholder="Node Name"></input>
-        <button id="search-button">Find Node</button>
-        <label id="node-error" class="error-text"></label>
+      <label>Zoom on Node in Graph: </label>
```

Comment:
can you add something about the behavior regarding zooming in on a node and a token? (is that allowed?)